snippets/core/system-text-json/csharp/Utf8ReaderFromFile.cs

```diff
@@ -36,11 +34,11 @@ public static void Run()
                         total++;
```

Comment:
Unrelated to this change, but the following ctor is more usable/approachable, given the scenario:

```diff
- var json = new Utf8JsonReader(jsonReadOnlySpan, isFinalBlock: true, state: default);
+ var json = new Utf8JsonReader(jsonReadOnlySpan);
```